test/index.spec.ts

```diff
@@ -304,4 +304,36 @@ describe('resin-semver', () => {
 			expect(semver.lt('Resin OS 2.0.0.dev', '2.0.0.dev')).to.equal(false);
 		});
 	});
+
+	describe('.satisfies()', () => {
+		const testRange = '^2.0.0';
+		it('should correctly evaluate valid semver values', () => {
+			expect(semver.satisfies('2.0.5', testRange)).to.equal(true);
+			expect(semver.satisfies('1.16.0', testRange)).to.equal(false);
+		});
+
+		it('should correctly evaluate Resin formatted versions', () => {
+			expect(semver.satisfies('Resin OS 2.0.2+rev2', testRange)).to.equal(true);
+			expect(semver.satisfies('Resin OS 2.0.2 (prod)', testRange)).to.equal(true);
+			expect(semver.satisfies('Resin OS 1.16.0', testRange)).to.equal(false);
+		});
+
+		it('should correctly evaluate invalid semver values', () => {
+			expect(semver.satisfies('Linux 14.04', testRange)).to.equal(false);
+			expect(semver.satisfies('A development version', testRange)).to.equal(false);
+			expect(semver.satisfies('Version A', testRange)).to.equal(false);
+		});
+
+		it('should correctly evaluate null values', () => {
+			expect(semver.satisfies(null, testRange)).to.equal(false);
```

Comment:
For the null/non-semver cases, I'd also include a quick couple of surprising cases (`*`, `< 1.0.0`), as both documentation and protection against accidental changes in this behaviour later, since that seems especially likely with debatable semantics like these.